package.json

```diff
@@ -7,6 +7,7 @@
     "@material-ui/icons": "^4.9.1",
     "@sage-bionetworks/react-base-table": "^1.13.2",
     "lodash-es": "^4.17.21",
+    "moment": "^2.29.4",
```

Comment:
I see--we were using moment and it Just Worked because SRC used to pull it in. Now that SRC uses dayjs, we need to manually specify it.

Reply:
That's correct